starlight/static/css/main.css

```diff
@@ -0,0 +1,28 @@
+@import url('https://fonts.googleapis.com/css?family=Proxima+Nova');
+
+body {
+  /*background: url('/static/images/starlight-background.png');*/
+  background-color: lightgray;
+  background-size: 100% auto;
+  font-family: 'proxima-nova';
```

Comment:
Nice choice :) Generally when specifying a font family in CSS you should provide additional system options as fallback in case that font cannot be loaded for whatever reason:

`font-family: 'Proxima Nova', 'Helvetica', sans-serif;`